package.json

```diff
@@ -41,20 +41,20 @@
     "@types/jest": "^24.0.17",
     "@types/node": "^12.7.0",
     "dtslint": "github:gcanti/dtslint",
-    "fast-check": "^1.16.0",
-    "fp-ts": "^2.0.3",
-    "io-ts": "^2.0.1",
+    "fast-check": "^2.10.0",
+    "fp-ts": "^2.9.3",
+    "io-ts": "^2.2.13",
```

Comment:
Note also that io-ts has deviated from semantic versioning above 2.1.3 :confused: 
everything above 2.1.3 and below 3 is called "experimental"".

Not sure that this will play nicely with users still on a "stable" version, but I'm not sure of what the best approach is here. O how I wish io-ts had used a dist-tag for the experimental work!

Reply:
> Note also that io-ts has deviated from semantic versioning above 2.1.3

Not really, meaning that 2.1 and above introduce experimental features without breaking the old/stable API. The current stable API will eventually be dropped in favor of the experimental one in a breaking release (3 I assume)

Reply:
My team have been experiencing compatibility issues between packages using different versions of io-ts across the 2.1.3 boundary. Our solution was to standardize on `~2.1.3` until `3.0.0` is released.

I believe the problem comes from providing a codec with a version of io-ts from one side of the boundary, and using that codec in a codec defined by io-ts on the other side of the boundary. Not sure if this library is at risk then :thinking: need to look at the internals more closely